CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - remove unnecessary horizontal scrollbar in "View Group" dialog #4254
 - change title of EditAccountAndPasswordDialog to make it clearer that it is about email account #4271
 - fix styling of progressbars in light theme #4274
+- add `libasound2` to debian package dependencies #4275
```

Comment:
```suggestion
- fix Delta Chat not launching on Debian sometimes due to missing package dependencies (`libasound2`) #4275
```

Wouldn't this be better for the target audience of `CHANGELOG.md`? This is more of a question from me.